idp/api/configuration/cloud/boldbi/mssql/sql_tables.sql

```diff
@@ -0,0 +1,1626 @@
+------------------------------------------------------------------
```

Comment:
Remove these folders

* idp/api/configuration/cloud
* idp/web/configuration/cloud
* idp/ums/configuration/cloud

Reply:
@r-rameshkannan Removed the mentioned folders.